modules/manage/pages/security/authorization/acl.adoc

```diff
@@ -55,7 +55,19 @@ When specifying hosts, you must pair the `--allow-host` flag with the `--allow-p
 
 === Resources
 
-A resource is what an ACL allows or denies access to. There are four resources within Redpanda: `topics`, `groups`, `cluster`, and `transactionalid`. Names for each of these resources can be specified with their respective flags.
+A resource is what an ACL allows or denies access to. The following resources are available within Redpanda: 
+
+- `topics`
+- `groups`
+- `cluster`
+- `transactionalid`
+
+Starting in version 25.2, Redpanda also supports the following resources for Schema Registry:
+
+- `registry-global` (for "top level" operations, such as `GET :8081/mode`, `GET :8081/config`.)
+- `registry-subject` (for "subject-level" operations, such as `GET :8081/subjects`, `GET :8081/config/{subject}`))
+
+Names for each of these resources can be specified with their respective flags.
 
 Resources combine with the operation that is allowed or denied on that resource. By default, resources are specified on an exact name match (a "literal" match).
```

Comment:
Should we also update the text below: https://docs.redpanda.com/current/manage/security/authorization/acl/#producingconsuming with the newer operations added in the rpk help text

(`rpk security acl --help-operations` has the new text)